src/components/Header/HeaderButton.ts

```diff
@@ -1,15 +1,18 @@
 import { CLASS } from '../../constants';
 import { Button } from '../../data/image';
-import Modal from '../Modal';
+import Modal from '../AddModal';
 
 const HeaderButton = {
   template() {
     return `
-      <button type="button" class="${CLASS.GNB_BUTTON}" aria-label="음식점 추가">
+      <button type="button" class="${CLASS.GNB_BUTTON}" aria-label="음식점" data-testId="headerButton">
```

Comment:
되도록이면 HTML attribute 이름은 kebab-case를 유지해주세요.
```suggestion
      <button type="button" class="${CLASS.GNB_BUTTON}" aria-label="음식점" data-test-id="headerButton">
```